src/test/scala/com/databricks/spark/xml/util/TypeCastSuite.scala

```diff
@@ -68,6 +68,9 @@ final class TypeCastSuite extends FunSuite {
     assert(TypeCast.castTo("1.00", FloatType, options) === 1.0)
     assert(TypeCast.castTo("1.00", DoubleType, options) === 1.0)
     assert(TypeCast.castTo("true", BooleanType, options) === true)
+    assert(TypeCast.castTo("1", BooleanType, options) == true)
```

Comment:
Tiny nit: use `===` to trigger the scalatest macros that print better errors when this fails

Reply:
Fixed.